index.js

```diff
@@ -70,6 +70,9 @@ function command(options) {
     cmd.push('echo "' + options.contents + '" >> ' + options.filename);
   }
 
-  cmd.push('git add .', 'git commit -m "' + options.message + '"');
+  if (!options.skipCommit) {
+    cmd.push('git add .', 'git commit -m "' + options.message + '"');
```

Comment:
I do wonder if maybe we want to split up the two commands since `git add .` doesn't exactly fall under `skipCommit`?

Reply:
I trust your judgment, I'll add you to the repository so you can merge and publish when you feel like it's good. thanks!

Reply:
done, added you to both this repository and as an owner on npm. thanks again!

Reply:
Thank you! 💟

Reply:
you are very welcome!